include/sapi/tss2_tcti.h

```diff
@@ -1,5 +1,5 @@
 //**********************************************************************;
```

Comment:
s/exstract/extract in the commit message.

Reply:
this one I can't fix since it's been merged, I'm working on the others though